### PR TITLE
virt: detect QNX hypervisor

### PR DESCRIPTION
**Pull request by Shuangistan**
Detect QNX hypervisor based on the CPUID.

Signed-off-by: Shuang Liu <sliu@de.adit-jv.com>

Fixes: #7239

### CODE REVIEW

File: man/systemd-detect-virt.xml [hunk 1]
```diff
@@ -143,11 +143,16 @@
...
-           <row>
+          <row>
             <entry><varname>bhyve</varname></entry>
             <entry>bhyve, FreeBSD hypervisor</entry>
           </row>
 
+          <row>
+            <entry><varname>qnx</varname></entry>
+            <entry>QNX hypervisor</entry>
+          </row>
```
Comment (poettering) [inline man/systemd-detect-virt.xml:154]: hmm, could you also update systemd.unit(5) please, there's a brief list of supervisor detected in the description of `ConditionVirtualization=`

**Comment by poettering** [comment]
Looks pretty OK, just one more tweak, please

**Comment by poettering** [comment]
btw, we do not use Signed-of-by on systemd. While we don't mind too much if its included we generally don't add it.

**Comment by poettering** [comment]
Oh, and please include "Fixes: #7239" in the commit msg, so that that issue is automatically closed when this PR is merged!

File: man/systemd-detect-virt.xml (hunk 1 above)
Comment (Shuangistan) [reply man/systemd-detect-virt.xml:154]: Thanks, done.

**Comment by poettering** [comment]
exellent! thanks!

**Pull request merged**